staging/src/k8s.io/kms/go.mod

```diff
@@ -19,11 +20,14 @@ require (
 	golang.org/x/net v0.7.0 // indirect
 	golang.org/x/sys v0.5.0 // indirect
 	golang.org/x/text v0.7.0 // indirect
+	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8 // indirect
 	google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 )
 
 replace (
+	k8s.io/api => ../api
```

Comment:
```suggestion
```

I cloned the PR to check and was able to remove this without any issues. Can you confirm if this was added because it was required? If not, we can remove it.

Reply:
Since the verify job hasn't been failing from what I can tell, this is likely required.

Reply:
confirmed it's being added when running `hack/update-vendor.sh`.

Reply:
This comment is resolved!